Assets/Scripts/Combat/CombatEntity.cs

```diff
@@ -122,9 +122,9 @@ public void GetAttacked(int rawDamage)
             InternalCurrentHealth = currentHealth - (rawDamage - currentDefense);
         }
 
-        public void GetDecreased(int rawDamage)
+        public void DecreasePlayerHealth(int hpDecreasingValue)
```

Comment:
Nice details! But maybe got too specific. This component is `CombatEntity`, so there is no guarantee that the entity will be the player character. We are adding this method because we are trying to decrease the player health, but who knows if we'll need it later on for some other purposes? `DecreaseHealth` would be enough. :3